pvlib/transformer.py

```diff
@@ -0,0 +1,119 @@
+"""
+This module contains functions for transformer modeling.
+
+Transformer models calculate AC power output at a different voltage from the
+voltage at the AC input terminals.
+"""
+
+import numpy as np
+
+
+def simple_efficiency(
+        input_power, no_load_loss, load_loss, transformer_rating
+):
+    r'''
+    Calculate the energy at the output terminal of the transformer
+    after taking into account efficiency using a simple calculation.
+
+    The equation used in this function can be derived from [1]_.
+
+
+    Parameters
+    ----------
+    input_power : numeric
+        The real power input to the transformer. [W]
+
+    no_load_loss : numeric
+        The constant losses experienced by a transformer, even
+        when the transformer is not under load. Fraction of transformer rating,
+        value from 0 to 1. [unitless]
+
+    load_loss:  numeric
+        The load dependent losses experienced by the transformer.
+        Fraction of transformer rating, value from 0 to 1. [unitless]
+
+    transformer_rating: numeric
+        The nominal output power of the transformer. [VA]
+
+    Returns
+    -------
+    output_power : numeric
+        Real power output. [W]
+
+    Notes
+    -------
+    First, assume that the load loss is proportional to the square of output
+    power.
+
+        .. math::
+
+           L_{load}(P_{out}) = L_{load}(P_{out}) * P^2_{out}
+           L_{load}(P_{out}) = L_{full, load} * P^2_{out}
```

Comment:
```suggestion
           L_{load}(P_{out}) = L_{load}(P_{out}) \times P^2_{out}

           L_{load}(P_{out}) = L_{full, load} \times P^2_{out}
```

I think a blank line is needed between equations.